src/components/accessories/patientOPD/patientOPDForm/PatientOPDForm.tsx

```diff
@@ -162,6 +171,20 @@ const PatientOPDForm: FunctionComponent<TProps> = ({
     [setFieldValue, handleBlur]
   );
 
+  const handleCheckboxChange = useCallback(
+    (fieldName: string) => (value: boolean) => {
+      setFieldValue(fieldName, value === true ? "R" : "");
+      if (value) {
+        if (fieldName === "referralFrom") {
+          setFieldValue("referralTo", "");
+        } else {
+          setFieldValue("referralFrom", "");
+        }
+      }
```

Comment:
You could simplify the condition like this:
`
if (value) {
  setFieldValue(fieldName, "");
}
`